src/prefect/futures.py

```diff
@@ -469,7 +636,7 @@ def _collect_futures(
         expr,
         visit_fn=partial(_collect_futures, futures),
         return_data=False,
-        context={},
+        context={"annotation": None},
```

Comment:
why do we have these?

Reply:
I was trying to fix some type errors. I'll revert.